src/bio2bel/sources/intact.py

```diff
@@ -292,8 +294,7 @@ def _add_my_row(graph: BELGraph, row) -> None:  # noqa:C901
                 )
                 continue
             # isomerase reaction
-            elif relation == 'isomerase reaction' \
-                    or relation == 'isomerization  reaction':
+            elif relation in {'isomerase reaction', 'isomerization  reaction'}:
```

Comment:
it looks like there are lots of inconsistencies in the relations, like double spaces. wouldn't it make it easier if you just mapped all double spaces to single spaces with a string replace at the beginning?

Reply:
aggreed, closed here d008a84c847650c573cf773657bdd2b48797a0fd